### DISCUSSION TITLE
Health bar on player

### DISCUSSION THREAD

**Discussion by aloranetwork** [zeshan321/ActionHealth#28] [Q&A]
hey i need help on how to take off the "Health : 10" off a players head in the settings

**Comment by zeshan321** [comment]
ActionHealth doesn't modify player names nor does it add anything to player heads. Are you using another health plugin (i.e mcmmo)?

**Reply by aloranetwork** [reply]
Na ever since i put the plugin under the players name it says “20 Health” and i want to know how to modify it that it doesn’t show

**Reply by zeshan321** [reply]
Screenshot please? Also, try removing the plugin and see if it still occurs. Like I said, ActionHealth only touches the action bar, nothing related to players heads/name.
